configs/typescript/tsconfig.basic.json

```diff
@@ -14,10 +14,6 @@
     "allowSyntheticDefaultImports": true /* Allow default imports from modules with no default export. This does not affect code emit, just typechecking. */,
     "esModuleInterop": true /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */,
     "resolveJsonModule": true /* Enable importing .json files." */,
-    "importsNotUsedAsValues": "error" /* Specify emit/checking behavior for imports that are only used for types */,
-    "typeRoots": [
-      "../node_modules/@types",
-      "../types"
-    ] /* List of folders to include type definitions from. */
+    "importsNotUsedAsValues": "error" /* Specify emit/checking behavior for imports that are only used for types */
```

Comment:
Is this really necessary to keep an explanation of every line in the configuration file? I think when in doubt about some rules we can read the [TypeScript docs](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html). What do you think about it?

Reply:
Yes, I agree, I just copy/pasted the contents from the other existing files and as I did not think it would hurt to have the comments there, I kept it but I believe now it is better to remove them. Let's see if the others also agree as well, but my vote is to remove them. Thanks!

Reply:
Yeah, I'd also say to remove them. If. there is any doubt, we go to the documentation. The same applies to the properties that are not there 😄 

_(applicable to all config files)_

Reply:
Removed! Thanks guys!